packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -224,13 +216,13 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
           {
             backgroundColor: globalTokens.color.grey[14],
             backgroundColorDark: globalTokens.color.white,
-            colorDark: globalTokens.color.grey[14],
+            colorDark: t.colors.neutralForeground1,
```

Comment:
Should be neutralBackground1 per redlines. It's grey 16 instead of 14 but it's what's specified in the table and web uses it too.

Reply:
Yeah, I wrote about this case in the comment to the PR, that I was waiting for the answer from the designer. But, yeah, it's better to change it according to the design